modelplugin/aether-2.0.0/README.md

```diff
@@ -0,0 +1,5 @@
+# YANG files
```

Comment:
could just delete this file since the yang files used to generate are "documented" in generator.sh I guess